src/qml/qgismobileapp.qml

```diff
@@ -2553,8 +2575,8 @@ ApplicationWindow {
     id: busyIndicator
     anchors.left: mainMenuBar.left
     anchors.top: mainToolbar.bottom
-    width: mainMenuBar.height
-    height: mainMenuBar.height
+    width: menuButton.width + 10
```

Comment:
Do we ever want a different aspect ratio than 1:1?

Reply:
@m-kuhn , not in this specific instance no.

Reply:
so rather set `width: height`?

Reply:
I'm doing height: width on the next line. Am I misunderstanding you?

Reply:
🫥 my mind must have still been asleep 🫥
Sorry!

Reply:
:)